.github/workflows/continuous-integration.yml

```diff
@@ -28,7 +28,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install kubernetes pyyaml requests jinja2 pystache pylint responses
+        pip install -r requirements.txt
+        pip install -r requirements_dev.txt
```

Comment:
it's better to use `extras_require` in `setup.py` if you want multiple envs

Reply:
Since we don't want to distribute this as a python package, keep using `requirements.txt`